english/octopus/new-octopus.txt

```diff
@@ -0,0 +1 @@
+That new octopus has eight forks and eats noodles eight times faster than a man.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.